### PR TITLE
Gho 1/doc/resume

### PR DESCRIPTION
**Pull request by alecio-rocha**
Proposed changes to the documentation:
- process to create a new branch
- summary of the day 12/24/2020

### CODE REVIEW

**Review by marcpires — CHANGES_REQUESTED** [review]
Fix the Markdown formatting as all text is as H2 (##). Just let the section head  formatted as H2

**Review by marcpires — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -8,7 +8,14 @@ Git and Bash hands on
...
-## =========== inicializando o repositório ======================
+## create a new branch
+
+```shell
+$ git checkout -b <name>
+```
+
+## =========== RESUMO DIA 24/12/2020 =======================================
+## =========== inicializando o repositório =================================
 ## Começamos inicializando um repositório para controle de versão utilizando
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply README.md]: I already solve

File: README.md [hunk 2]
```diff
@@ -17,7 +24,7 @@ $ sudo apt install -y git-all
 
 ## ======= trabalhando com o controle remoto ==================
 ## Começamos agora a gerenciar o repositório de forma remota, onde primeiro
```
[the remark this answers is not in the document]
Comment (alecio-rocha) [reply README.md]: I already solve

**Pull request merged**